Zenodo links in README resolve in web browser but not in CI testing

https://github.com/openforcefield/openff-toolkit/actions/runs/21051534538/job/60538725907#step:14:27

```
_ test_readme_links[https://zenodo.org/badge/DOI/10.5281/zenodo.17525527.svg] __
openff/toolkit/_tests/test_links.py:73: in test_readme_links
    raise exception
openff/toolkit/_tests/test_links.py:67: in test_readme_links
    urlopen(request)
../../../micromamba/envs/openff-toolkit-test/lib/python3.11/urllib/request.py:216: in urlopen
    return opener.open(url, data, timeout)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
../../../micromamba/envs/openff-toolkit-test/lib/python3.11/urllib/request.py:525: in open
    response = meth(req, response)
               ^^^^^^^^^^^^^^^^^^^
../../../micromamba/envs/openff-toolkit-test/lib/python3.11/urllib/request.py:634: in http_response
    response = self.parent.error(
../../../micromamba/envs/openff-toolkit-test/lib/python3.11/urllib/request.py:563: in error
    return self._call_chain(*args)
           ^^^^^^^^^^^^^^^^^^^^^^^
../../../micromamba/envs/openff-toolkit-test/lib/python3.11/urllib/request.py:496: in _call_chain
    result = func(*args)
             ^^^^^^^^^^^
../../../micromamba/envs/openff-toolkit-test/lib/python3.11/urllib/request.py:643: in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
E   urllib.error.HTTPError: HTTP Error 403: Forbidden
```

--- a/README.md
+++ b/README.md
@@ -21,13 +21,9 @@ The toolkit currently covers two main areas we have committed to stably maintain
 
 ## How to cite
 
-Please cite the OpenFF Toolkit using the [Zenodo record](https://zenodo.org/record/13886754) of the [latest release](https://zenodo.org/record/13886754) or the version that was used. The BibTeX reference of the latest release can be found [here](https://zenodo.org/records/13886754/export/bibtex).
+Please cite the OpenFF Toolkit using the Zenodo record of the [latest release](https://zenodo.org/records/13886754/latest) or the version that was used. 
 
 ## Installation
-
-The Open Force Field Toolkit (`openff-toolkit`) is a Python toolkit, and supports Python 3.9 through 3.11.
-
-### Installing via Mamba/Conda
 
 Detailed installation instructions can be found [here](https://open-forcefield-toolkit.readthedocs.io/en/stable/installation.html).
 

--- a/openff/toolkit/_tests/test_links.py
+++ b/openff/toolkit/_tests/test_links.py
@@ -49,11 +49,16 @@ def test_readme_links(readme_link):
         pytest.skip("DOI links are behind DDoS protection and do not resolve")
     if "codecov.io" in readme_link:
         pytest.skip("Codecov website DOI also may be behind DDoS protection")
-    if "export" in readme_link:
-        # October 2023 Zenodo upgrade seems to have broken this link - although
-        # it is still accessible from the web UI (go to the landing page of a
-        # record, Export section (select BibTeX in drop-down) and click "Export"
+    if "zenodo" in readme_link:
+        # October 2023 Zenodo upgrade seems to have broken the bibtex export
+        # link although it is still accessible from the web UI (go to the
+        # landing page of a record, Export section (select BibTeX in
+        # drop-down) and click "Export"
         # button will bring you to this link.
+
+        # Jan 2026 Zenodo seems to have blocked GitHub Actions IP access to
+        # regular release pages as well, so skip all zenodo checks.
+        # https://support.zenodo.org/help/en-gb/13-policies/29-why-is-my-ip-address-blocked
         pytest.skip("Seems to be a Zenodo regression in bibtex link")
     if readme_link.endswith("MIT"):
         # January 22 2024: Link causing some failures, accessible in browser

--- a/openff/toolkit/_tests/test_links.py
+++ b/openff/toolkit/_tests/test_links.py
@@ -53,13 +53,12 @@ def test_readme_links(readme_link):
         # October 2023 Zenodo upgrade seems to have broken the bibtex export
         # link although it is still accessible from the web UI (go to the
         # landing page of a record, Export section (select BibTeX in
-        # drop-down) and click "Export"
-        # button will bring you to this link.
+        # drop-down) and click "Export", will work in browser)
 
-        # Jan 2026 Zenodo seems to have blocked GitHub Actions IP access to
-        # regular release pages as well, so skip all zenodo checks.
+        # Jan 2026 Zenodo seems to have blocked programmatic (non-browser)
+        # access to regular release pages as well, so skip all zenodo checks.
         # https://support.zenodo.org/help/en-gb/13-policies/29-why-is-my-ip-address-blocked
-        pytest.skip("Seems to be a Zenodo regression in bibtex link")
+        pytest.skip("Programmatic access tests no longer work for Zenodo URLs")
     if readme_link.endswith("MIT"):
         # January 22 2024: Link causing some failures, accessible in browser
         pytest.skip("Flaky")

